modules/post/windows/gather/checkvm.rb

```diff
@@ -202,7 +260,9 @@ def report_vm(hypervisor)
   def run
     print_status('Checking if the target is a Virtual Machine ...')
 
-    if hyperv?
+    if parallels?
+      report_vm('Parallels')
+    elsif hyperv?
       report_vm('Hyper-V')
     elsif vmware?
       report_vm('VMware')
```

Comment:
My Windows 11 VM running inside VMware that had Hyper-V enable was being flagged as running inside Hyper-V. Rearranging these solved the issue.
```suggestion
    elsif vmware?
      report_vm('VMware')
    elsif hyperv?
      report_vm('Hyper-V')
```